src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  booksReducer,
+  categoriesReducer,
+});
+
+const store = createStore(rootReducer);
+
```

Comment:
- [ ] [Optional] It would be nice to import `applyMiddleware` from `redux` to use some handy `Redux` middleware, like a redux logger. If you use the logger you would see the previous state, and current state and also understand the inner workings of redux